### ISSUE TITLE
Error in RenameFiles WOH

### ISSUE THREAD

**Issue by rbeltranb** [opencast/opencast#3489]
**Describe the bug**

Using a "rename-files" operation at the end of the 'publish-after-cutting' workflow, between the "snapshot" and "cleanup" operations:

```xml
    <operation
      id="rename-files"
      description="Rename files based on metadata"
      <configurations>
        <configuration key="name-pattern">#{episode.title}_#{episode.creator}.#{file.extension}</configuration>
        <configuration key="source-flavors">*/*</configuration>
      </configurations>
    </operation>
```

**To Reproduce**
After deploying the modified "publish-after-cutting.xml" file:
1. Open the Video Editor for a selected Event 
2. Mark two points for 'Split'
3. Hide the middle segment
4. Select the workflow 'Publish', which corresponds to the deployed XML
5. Click on 'Save & Continue'.

**Environment Information**
 - OS: Scientific Linux 7.9 (Nitrogen)
 - Browser: Safari
 - Opencast version: r/10.x

**Error**
```
2022-02-23 17:26:44,461 \| INFO  \| (RenameFilesWorkflowOperationHandler:101) - Running rename files workflow operation on workflow 23436
2022-02-23 17:26:44,525 \| INFO  \| (RenameFilesWorkflowOperationHandler:139) - Renaming https://localhost/assets/assets/a5ec1136-07a0-475b-9b49-eb8dcb96c0f5/abaa6b55-65b5-4d19-9a18-e8cbae315380/30/Matthias_lecture_trimmed.mp4 to https://localhost/files/mediapackage/a5ec1136-07a0-475b-9b49-eb8dcb96c0f5/7a753c65-bc30-4a7f-b254-23c46ff5db95/Fast_workflow_with_copy_12_raub.mp4
2022-02-23 17:26:44,675 \| INFO  \| (RenameFilesWorkflowOperationHandler:139) - Renaming https://localhost/assets/assets/a5ec1136-07a0-475b-9b49-eb8dcb96c0f5/8abb6e92-6bcf-4c1f-af04-5c3d6b6b5eb3/30/8abb6e92_6bcf_4c1f_af04_5c3d6b6b5eb3.mp4 to https://localhost/files/mediapackage/a5ec1136-07a0-475b-9b49-eb8dcb96c0f5/1efdb8d9-8fab-4ccb-a95e-fc263ba030cf/Fast_workflow_with_copy_12_raub.mp4
2022-02-23 17:26:44,832 \| INFO  \| (RenameFilesWorkflowOperationHandler:139) - Renaming https://localhost/assets/assets/a5ec1136-07a0-475b-9b49-eb8dcb96c0f5/8abb6e92-6bcf-4c1f-af04-5c3d6b6b5eb3/30/8abb6e92_6bcf_4c1f_af04_5c3d6b6b5eb3.mp4 to https://localhost/files/mediapackage/a5ec1136-07a0-475b-9b49-eb8dcb96c0f5/9720b65f-8bba-4b3a-81d0-b13b6a2ceb60/Fast_workflow_with_copy_12_raub.mp4
2022-02-23 17:26:44,886 \| INFO  \| (RenameFilesWorkflowOperationHandler:139) - Renaming https://localhost/files/mediapackage/a5ec1136-07a0-475b-9b49-eb8dcb96c0f5/eed1f948-696a-4a47-a0b4
[workflow-23505-23570-med
[workflow-23505-23570-mediapackage-a5ec1136-07a0-475b-9b49-eb8dcb96c0f5.log](https://github.com/opencast/opencast/files/8126928/workflow-23505-23570-mediapackage-a5ec1136-07a0-475b-9b49-eb8dcb96c0f5.log)
iapackage-a5ec1136-07a0-475b-9b49-eb8dcb96c0f5.log](https://github.com/opencast/opencast/files/8126926/workflow-23505-23570-mediapackage-a5ec1136-07a0-475b-9b49-eb8dcb96c0f5.log)
-adbb339685e3/presenter_eed1f948_696a_4a47_a0b4_adbb339685e3.mp4 to https://localhost/files/mediapackage/a5ec1136-07a0-475b-9b49-eb8dcb96c0f5/a45e3d11-0447-40ad-94ab-b80e6a6cb80a/Fast_workflow_with_copy_12_raub.mp4
2022-02-23 17:26:44,890 \| WARN  \| (WorkspaceImpl:400) - The working file repository URI and paths don't match. Looking up https://localhost/files/mediapackage/a5ec1136-07a0-475b-9b49-eb8dcb96c0f5/eed1f948-696a-4a47-a0b4-adbb339685e3/presenter_eed1f948_696a_4a47_a0b4_adbb339685e3.mp4 at /srv/opencast/files/mediapackage/a5ec1136-07a0-475b-9b49-eb8dcb96c0f5/eed1f948-696a-4a47-a0b4-adbb339685e3/presenter_eed1f948_696a_4a47_a0b4_adbb339685e3.mp4 failed
2022-02-23 17:26:44,891 \| WARN  \| (WorkspaceImpl:368) - The working file repository and workspace paths don't match. Looking up https://localhost/files/mediapackage/a5ec1136-07a0-475b-9b49-eb8dcb96c0f5/eed1f948-696a-4a47-a0b4-adbb339685e3/presenter_eed1f948_696a_4a47_a0b4_adbb339685e3.mp4 at /srv/opencast/files/mediapackage/a5ec1136-07a0-475b-9b49-eb8dcb96c0f5/eed1f948-696a-4a47-a0b4-adbb339685e3/presenter_eed1f948_696a_4a47_a0b4_adbb339685e3.mp4 failed

2022-02-23 17:26:44,930 \| ERROR \| (WorkflowOperationWorker:141) - Workflow operation 'operation:'rename-files', position:43, state:'FAILED'' failed

org.opencastproject.util.NotFoundException: null
at org.opencastproject.workspace.impl.WorkspaceImpl.downloadIfNecessary(WorkspaceImpl.java:516) ~[?:?]
at org.opencastproject.workspace.impl.WorkspaceImpl$2.xapply(WorkspaceImpl.java:538) ~[?:?]
at org.opencastproject.workspace.impl.WorkspaceImpl$2.xapply(WorkspaceImpl.java:535) ~[?:?]
at org.opencastproject.util.data.Function$X.apply(Function.java:104) ~[!/:?]
at org.opencastproject.util.IoSupport.locked(IoSupport.java:507) ~[!/:?]
at org.opencastproject.workspace.impl.WorkspaceImpl.get(WorkspaceImpl.java:383) ~[?:?]
at org.opencastproject.workspace.impl.WorkspaceImpl.read(WorkspaceImpl.java:418) ~[?:?]
at org.opencastproject.workflow.handler.rename.RenameFilesWorkflowOperationHandler.start(RenameFilesWorkflowOperationHandler.java:137) ~[?:?]
```

**Log of the workflow status:**

```json
{
    "mediapackage": {
        "duration": 11923,
        "id": "c02b9409-decd-4b46-9ba2-f09d486b7ec6",
        "start": "2022-01-10T11:42:10Z",
        "title": "raub_test_1_10012021",
        "media": {
            "track": {
                "id": "df429afb-3b16-455f-b172-03e962e0d327",
                "type": "presenter/source",
                "mimetype": "video/mp4",
                "tags": {
                    "tag": "archive"
                },
                "url": "https://localhost:8080/assets/assets/c02b9409-decd-4b46-9ba2-f09d486b7ec6/df429afb-3b16-455f-b172-03e962e0d327/4/KliMB_Training_11_30_2021_2016_34_13_1.mp4",
                "size": 45471045,
                "checksum": {
                    "type": "md5",
                    "$": "21288ac6849680c0adbc3d85de257e2a"
                },
                "duration": 11923,
                "audio": {
                    "id": "audio-1",
                    "device": "",
                    "encoder": {
                        "type": "AAC (Advanced Audio Coding)"
                    },
                    "framecount": 515,
                    "channels": 2,
                    "samplingrate": 44100,
                    "bitrate": 9269
                },
                "video": {
                    "id": "video-1",
                    "device": "",
                    "encoder": {
                        "type": "H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10"
                    },
                    "framecount": 714,
                    "bitrate": 3.049147E7,
                    "framerate": 59.882584,
                    "resolution": "1440x1920"
                },
                "live": false,
                "master": false
            }
        },
        "metadata": {
            "catalog": {
                "id": "94d5ac0f-6832-4724-8c74-0167cfd5f93c",
                "type": "dublincore/episode",
                "mimetype": "text/xml",
                "tags": {
                    "tag": "archive"
                },
                "url": "https://localhost:8080/assets/assets/c02b9409-decd-4b46-9ba2-f09d486b7ec6/94d5ac0f-6832-4724-8c74-0167cfd5f93c/4/dublincore.xml",
                "checksum": {
                    "type": "md5",
                    "$": "724f6cc1db7510547eef744892ed65dd"
                }
            }
        },
        "attachments": {
            "attachment": {
                "id": "security-policy-episode",
                "type": "security/xacml+episode",
                "mimetype": "text/xml",
                "tags": {
                    "tag": "archive"
                },
                "url": "https://localhost:8080/assets/assets/c02b9409-decd-4b46-9ba2-f09d486b7ec6/security-policy-episode/4/xacml.xml",
                "checksum": {
                    "type": "md5",
                    "$": "9569316296835d4f7fb7b88e521fb793"
                }
            }
        },
        "publications": {
            "publication": [
                {
                    "id": "2940fdd3-a34e-4fc9-b25c-1197a23ce7d9",
                    "channel": "internal",
                    "tags": {
                        "tag": "archive"
                    },
                    "url": "http://localhost:8080/admin-ng/index.html#/events/events/c02b9409-decd-4b46-9ba2-f09d486b7ec6/tools/playback",
                    "media": {
                        "track": {
                            "id": "02bb19b9-37c7-4c36-8bbf-af4f602532cb",
                            "type": "presenter/preview",
                            "ref": "track:df429afb-3b16-455f-b172-03e962e0d327",
                            "mimetype": "video/mp4",
                            "tags": {
                                "tag": [
                                    "atom",
                                    "default",
                                    "engage-download",
                                    "engage-streaming",
                                    "rss"
                                ]
                            },
                            "url": "https://localhost:8080/static/mh_default_org/internal/c02b9409-decd-4b46-9ba2-f09d486b7ec6/013a7d5b-986d-4f94-a0b5-769f014056cb/KliMB_Training_11_30_2021_2016_34_13_1.mp4",
                            "size": 14663969,
                            "checksum": {
                                "type": "md5",
                                "$": "8b876c1d4e2bd97a9c532b8579a6296e"
                            },
                            "duration": 11934,
                            "audio": {
                                "id": "audio-1",
                                "device": "",
                                "encoder": {
                                    "type": "AAC (Advanced Audio Coding)"
                                },
                                "framecount": 514,
                                "channels": 2,
                                "samplingrate": 44100,
                                "bitrate": 13652
                            },
                            "video": {
                                "id": "video-1",
                                "device": "",
                                "encoder": {
                                    "type": "H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10"
                                },
                                "framecount": 715,
                                "bitrate": 9806893,
                                "framerate": 59.94006,
                                "resolution": "1440x1920"
                            },
                            "live": false,
                            "master": false
                        }
                    },
                    "attachments": "",
                    "metadata": ""
                },
                {
                    "id": "deec9611-ef5d-4096-afe2-9716533bdd41",
                    "channel": "engage-player",
                    "mimetype": "text/html",
                    "tags": {
                        "tag": "archive"
                    },
                    "url": "https://localhost:8080/play/c02b9409-decd-4b46-9ba2-f09d486b7ec6",
                    "media": "",
                    "attachments": "",
                    "metadata": ""
                },
                {
                    "id": "c4867b54-5190-4c56-94bd-66f25518a9cf",
                    "channel": "api",
                    "mimetype": "application/json",
                    "tags": {
                        "tag": "archive"
                    },
                    "url": "http://localhost:8080/api/events/c02b9409-decd-4b46-9ba2-f09d486b7ec6",
                    "media": {
                        "track": {
                            "id": "ae6a5241-74f7-4a88-bd79-81f5e70c6710",
                            "type": "presenter/delivery",
                            "ref": "track:acf686ef-c263-479b-a544-44dc057e228a",
                            "mimetype": "video/mp4",
                            "tags": {
                                "tag": [
                                    "720p-quality",
                                    "default",
                                    "engage-download",
                                    "engage-streaming"
                                ]
                            },
                            "url": "https://localhost:8080/static/mh_default_org/api/c02b9409-decd-4b46-9ba2-f09d486b7ec6/382dbc60-2fa4-4223-a585-16cb49ac4b45/2ad57903_e7e8_4e73_ae4d_e81b0e9ed794.mp4",
                            "size": 1928541,
                            "checksum": {
                                "type": "md5",
                                "$": "bcfe802666fe541b893a37f0efe908f5"
                            },
                            "duration": 11934,
                            "audio": {
                                "id": "audio-1",
                                "device": "",
                                "encoder": {
                                    "type": "AAC (Advanced Audio Coding)"
                                },
                                "framecount": 514,
                                "channels": 1,
                                "samplingrate": 44100,
                                "bitrate": 7977
                            },
                            "video": {
                                "id": "video-1",
                                "device": "",
                                "encoder": {
                                    "type": "H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10"
                                },
                                "framecount": 298,
                                "bitrate": 1278681,
                                "framerate": 25,
                                "resolution": "1280x1706"
                            },
                            "live": false,
                            "master": false
                        }
                    },
                    "attachments": {
                        "attachment": [
                            {
                                "id": "77cd9b3b-f02f-4011-bca3-9f3276f40d74",
                                "type": "presenter/search+preview",
                                "ref": "track:94d1bc9a-9ccb-4bc8-9b8d-fa1d52f56402",
                                "mimetype": "image/jpeg",
                                "tags": {
                                    "tag": "engage-download"
                                },
                                "url": "https://localhost:8080/static/mh_default_org/api/c02b9409-decd-4b46-9ba2-f09d486b7ec6/9efdd46b-e284-422a-819f-ef55c008a0ce/2ad57903_e7e8_4e73_ae4d_e81b0e9ed794_1_000s_search.jpg",
                                "size": 11923
                            },
                            {
                                "id": "d33fa0e0-ef55-4ef3-810b-271dad43ae84",
                                "type": "presenter/timeline+preview",
                                "ref": "track:track-4",
                                "mimetype": "image/png",
                                "tags": {
                                    "tag": "engage-download"
                                },
                                "url": "https://localhost:8080/static/mh_default_org/api/c02b9409-decd-4b46-9ba2-f09d486b7ec6/7a603def-d094-4202-a039-f285f89cd08c/2ad57903_e7e8_4e73_ae4d_e81b0e9ed794_49d22e66_d4b6_4ce2_af8f_047d6ac78fab_timelinepreviews.png",
                                "additionalProperties": {
                                    "property": [
                                        {
                                            "key": "resolutionY",
                                            "$": "-1"
                                        },
                                        {
                                            "key": "imageCount",
                                            "$": "100"
                                        },
                                        {
                                            "key": "resolutionX",
                                            "$": "160"
                                        },
                                        {
                                            "key": "imageSizeY",
                                            "$": "10"
                                        },
                                        {
                                            "key": "imageSizeX",
                                            "$": "10"
                                        }
                                    ]
                                }
                            },
                            {
                                "id": "8a695723-f8a0-463a-a019-b3f79bfdd99b",
                                "type": "presenter/player+preview",
                                "mimetype": "image/png",
                                "tags": {
                                    "tag": [
                                        "archive",
                                        "engage-download"
                                    ]
                                },
                                "url": "https://localhost:8080/static/mh_default_org/api/c02b9409-decd-4b46-9ba2-f09d486b7ec6/3545f63a-933f-44b4-9dd3-e7fb98afd29b/coverimage.png"
                            }
                        ]
                    },
                    "metadata": ""
                }
            ]
        }
    }
}
```

**Comment by lkiesow** [comment]
GitHub seems to have scrambled your logs. Can you re-pate them in a code block:

    ```
    ...logs here...
    ```

**Comment by rbeltranb** [comment]
Error and workflow status logs updated.

**Comment by mliradelc** [comment]
While this go before the `Cleanup` operation, we can't discard that #3510 is involved on this issue, possible fix: #3541. 

Update: The mentioned "Fix" was introduced on 11.5, @rbeltranb, Can you check if solves the mentioned issue?

**Comment by lkiesow** [comment]
I did not manage to reproduce this issue with a current Opencast.
I did, however, notice that there is a syntax error in the configuration you provided.
The line starting with `description` should have a closing `>`.
Maybe that caused the issue.

Feel free to re-open this if you can still reproduce the issue.
If possible, attach the modified workflow.
